docs/fsharp/language-reference/namespaces.md

```diff
@@ -73,6 +73,52 @@ You can also use global to reference the top-level .NET namespace, for example,
 
 [!code-fsharp[Main](../../../samples/snippets/fsharp/lang-ref-2/snippet6408.fs)]
 
+##
+
+F# 4.1 introduces the notion of namespaces which allow for all underlying code to be mutually recursive.  This is done via `namespace rec`.  Use of `module rec` can alleviate some pains in not being able to write mutually referential code between types and modules.  The following is an example of this:
```

Comment:
Typo:
This is done via `namespace rec`.  Use of `module rec` ...
should be:
This is done via `namespace rec`.  Use of `namespace rec` ...